samples/Chat/src/app/EndScreen.tsx

```diff
@@ -70,6 +71,7 @@ export const EndScreen = (props: EndCallProps): JSX.Element => {
       className={endChatContainerStyle}
     >
       <Stack tokens={upperStackTokens}>
+        <Announcer announcementString={leftCall} ariaLive={'polite'} />
```

Comment:
This is good! Another more recommended way to do this would be to verify that the text in the heading is read instead of using an `announcer` just so we don't have a army of them running around in the DOM. you can set the aria-label to be what you gave the `announcer`. sometimes Fluent components (I'm assuming `Text` is from fluent) can have issues if their aria label is not set. their syntax for setting them is a prop like: `ariaLabel={'your string here'}` .

Reply:
> This is good! Another more recommended way to do this would be to verify that the text in the heading is read instead of using an `announcer` just so we don't have a army of them running around in the DOM. you can set the aria-label to be what you gave the `announcer`. sometimes Fluent components (I'm assuming `Text` is from fluent) can have issues if their aria label is not set. their syntax for setting them is a prop like: `ariaLabel={'your string here'}` .

Thanks Donald. Moved it to Text itself.